back-end/sports_platform/sports_platform_api/views/user_views.py

```diff
@@ -1,15 +1,36 @@
 from django.db.utils import IntegrityError
+from rest_framework.authtoken.models import Token
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
-from ..controllers.guest import Guest
-from ..validation import user_validation
-from django.contrib.auth.decorators import login_required
-from ..models import User
+
 from ..controllers import Guest
-import re
-from rest_framework.authtoken.models import Token
-import django.contrib.auth 
+from ..models import User
+from ..serializers.user_serializer import UserSerializer
+from ..validation import user_validation
+
 
+@api_view(['GET'])
+def get_user(request, user_id):
+    if not request.user.is_authenticated:
+        return Response(data={'message': 'User is not logged in, first you need to login'}, status=401)
+    try:
+        user = User.objects.get(pk=user_id)
+        sports = user.get_sport_skills()
+    except User.DoesNotExist:
+        return Response(data={"message": 'User id does not exist'}, status=400)
+    except Exception:
+        return Response(data={'message': 'An error occured, please try again later.'}, status=500)
+    
+    serialized_user = UserSerializer(user).data
+    # drop last_login field that we  do not use
+    serialized_user.pop('last_login')
+    # add activity stream data
+    serialized_user['@context'] = 'https://schema.org/Person'
+    serialized_user['@id'] = user.user_id
+    serialized_user['sports'] = sports
```

Comment:
For this field, we should make it consistent with the schema.

Reply:
I have changed it feel free to review and if it is ok, please resolve this